homeassistant/components/sensor/zwave.py

```diff
@@ -7,15 +7,27 @@
 at https://home-assistant.io/components/zwave.html
 """
 # pylint: disable=import-error
+from homeassistant.helpers.event import track_point_in_time
 from openzwave.network import ZWaveNetwork
 from pydispatch import dispatcher
-
+import datetime
+import homeassistant.util.dt as dt_util
 import homeassistant.components.zwave as zwave
 from homeassistant.helpers.entity import Entity
 from homeassistant.const import (
     ATTR_BATTERY_LEVEL, STATE_ON, STATE_OFF,
     TEMP_CELCIUS, TEMP_FAHRENHEIT, ATTR_LOCATION)
 
+PHILIO = '013c'
+PHILIO_SLIM_SENSOR = '0002'
+PHILIO_SLIM_SENSOR_MOTION = (PHILIO, PHILIO_SLIM_SENSOR, 0)
+
+WORKAROUND_NO_TRIGGER_OFF_EVENT = 'trigger_no_off_event'
+
+SPECIFIC_DEVICE_MAPPINGS = [
```

Comment:
Good idea! Will change this! 

Also as you saw I the xml mappings file, I can get a PIR time out value. I'm thinking that I will check this and add some extra time before invalidating the value, if it redirects motion we don't want to trigger an off event in between!